### PR TITLE
Form fields styling

### PR DESCRIPTION
**Pull request by Mister-42**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌
| Fixed issues  |

### CODE REVIEW

File: config/web/field.php [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?php
+
+declare(strict_types=1);
+
+use Composer\InstalledVersions;
+use Yiisoft\Form\Widget\Field;
+
+if (InstalledVersions::isInstalled('yiisoft/form') && $params['yiisoft/form']['bulma']['enabled'] === true) {
```
Comment (samdark) [inline config/web/field.php:8]: The condition looks odd. I don't think there's a need for it. Even if form package isn't installed, it should work alright.

File: composer.json [hunk 2]
```diff
@@ -24,6 +24,7 @@
         "npm-asset/bulma-helpers": "^0.3.12",
         "npm-asset/vizuaalog--bulmajs": "^0.11.0",
         "oomphinc/composer-installers-extender": "^2.0.0",
+        "yiisoft/composer-config-plugin": "^1.0",
```
Comment (samdark) [inline composer.json:27]: It isn't required for the  package to function. I'd remove it from here.

File: config/params.php [hunk 3]
```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+return [
+    'yiisoft/form' => [
+        'bulma' => [
+            'enabled' => true,
```
Comment (samdark) [inline config/params.php:8]: What's the use case for disabling it?

**Comment by samdark** [comment]
@terabytesoftw is overall concept alright?

File: config/web/field.php (hunk 1 above)
Comment (Mister-42) [reply config/web/field.php:8]: I was afraid of error if Form is not installed. So, not required?

File: config/params.php (hunk 3 above)
Comment (Mister-42) [reply config/params.php:8]: Multiple packages can be installed (yii-bulma for example). This makes sure the unwanted changes can be turned off.

Comment (Mister-42) [reply config/params.php:8]: @terabytesoftw insisted of it being able to be disabled

**Pull request merged**